sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/VerifyMigrationTask.kt

```diff
@@ -116,7 +124,13 @@ abstract class VerifyMigrationTask : SqlDelightWorkerTask() {
       if (!environment.dialect.isSqlite) return
       parameters.workingDirectory.get().asFile.deleteRecursively()
 
-      val catalog = createCurrentDb()
+      val connectionPropertiesMap = parameters.connectionProperties.get()
+      val connectionProperties = Properties()
+      connectionPropertiesMap.forEach { (key, value) ->
+        connectionProperties[key] = value
+      }
+      ServiceLoader.load(MigrationVerificationDriver::class.java)
```

Comment:
Initially I placed this call into `CatalogDatabase#createConnection(...)`, but that module isn't exposed to clients, so I wouldn't be able to access `MigrationVerificationDriver` interface on the consumer side in order to setup `ServiceLoader`